docs/config.md

```diff
@@ -258,26 +264,34 @@ objectives:
 The above config will multiply obtained throughput and latency for each
 measurement by 0.5 (i.e. 0.5 * latency + 0.5 * throughput) and use this as
 the "score" to sort the measurements according to. This mode can be useful
-if you have a more advanced criteria upon which you want to sort the results.
+if a more advanced criteria needs to be used for sorting.
 
 An extension of the above `objectives` is explicitly specifying the weights.
 For example:
 ```yaml
 objectives:
-    perf_latency: 10
-    perf_throughput: 15
+    perf_latency: 2
+    perf_throughput: 3
 ```
 
 The score for each measurement will be a weighted average using the weights specified here.
```

Comment:
You should mention that the options above will 40% (2/5) to perf_latency and 60% (3/5) to perf_throughput scores.